grails-app/services/au/org/ala/ecodata/ElasticSearchService.groovy

```diff
@@ -1093,7 +1093,7 @@ class ElasticSearchService {
             // GeoServer requires a single attribute with project area. Cannot use `sites` property (above) since it has
             // all sites associated with project.
             // todo: Check if BioCollect requires all sites in `sites` property. If no, merge `projectArea` with `sites`.
-            projectMap.projectArea = siteService.get(project.projectSiteId, [SiteService.FLAT, SiteService.INDEXING])
+            projectMap.projectArea = siteService.getSimpleProjectArea(projectMap.projectSiteId)
```

Comment:
If I'm understanding this correctly, the geoIndex here is being populated at index time.  Is it still a persistent property of the Site?  This will likely cause a side effect of updating every site during indexing as GORM is likely to mark the site as dirty and commit it on the transaction boundary. (I am not 100% sure of this, but I don't think GORM requires an explicit save()).
It also means we are storing the full resolution site and presumably rendering it in BioCollect?  Is this what we want or are we better off just simplifying the site as required on import and saving the simplified site?